src/UI/Core/ListBoxBase.cpp

```diff
@@ -125,7 +125,7 @@ void ListBoxBase::onMouseDown(EventHandler::MouseButton button, int x, int y)
 	// A few basic checks
 	if (!rect().contains(mMousePosition) || mCurrentHighlight == constants::NO_SELECTION) { return; }
 	if (mSlider.visible() && mSlider.rect().contains(Point{x, y})) { return; }
-	if (mCurrentHighlight < 0 || static_cast<size_t>(mCurrentHighlight) >= mItems.size()) { return; }
+	if (static_cast<size_t>(mCurrentHighlight) >= mItems.size()) { return; }
```

Comment:
As a matter of style, I would prefer `operator<`.

Reply:
You mean comparison against `size()`? I suppose I am more used to seeing `<` in that context. Though I suppose this one is inverted for an early return.